### PR TITLE
If no IOP is present, 'istioctl verify-install' detects Istio using Istio's webhook

### PR DESCRIPTION
**Pull request by esnible**
Resolves https://github.com/istio/istio/issues/29607

### CODE REVIEW

File: istioctl/pkg/verifier/verifier.go [hunk 1]
```diff
@@ -115,6 +116,16 @@ func (v *StatusVerifier) verifyInstallIOPRevision() error {
...
 	if err != nil {
+		// At this point we know there is no IstioOperator defining a control plane.  This may
+		// be the case in a Istio cluster with external control plane.
+		injector, err2 := v.injectorFromCluster(v.controlPlaneOpts.Revision)
+		if err2 == nil && injector != nil {
+			// The cluster *is* configured for Istio, but no IOP is present.  This could mean
+			// - the user followed our remote control plane instructions
+			// - helm was used
+			// - user did `istioctl manifest generate | kubectl apply ...`
+			return fmt.Errorf("cluster configured for Istio injection.  Supply flag --filename <yaml>")
```
Comment (shamsher31) [inline istioctl/pkg/verifier/verifier.go:127]: nit: extra space.
```suggestion
			return fmt.Errorf("cluster configured for Istio injection. Supply flag --filename <yaml>")
```

**Comment by shamsher31** [comment]
> The message SHOULD say something like "Remotely managed Istio data plane found."

Do you want to include a message that you mentioned in the issue?

**Comment by esnible** [comment]
> Do you want to include a message that you mentioned in the issue?

@shamsher31 Thanks for reviewing this.  It seems there is no way to tell if the user is missing an IOP because they are remote or for some other reason.  I made the error message more generic.  I believe that is the best choice at this time.  Unless I can convince Environments WG to create IOP for remote and helm-installed planes we can't be overly-specific about causes.

File: releasenotes/notes/29855.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+- 29067
```
Comment (ericvn) [inline releasenotes/notes/29855.yaml:5]:
```suggestion
- 29607
```

**Review by shamsher31 — APPROVED** [review]
Thanks.

**Pull request merged**